library/Fission/AWS/Types.hs

```diff
@@ -2,7 +2,14 @@
 module Fission.AWS.Types
   ( DomainName (..)
   , ZoneID (..)
+  , MockEnabled (..)
   ) where
 
 import Fission.AWS.DomainName.Types
 import Fission.AWS.Zone.Types
+import Fission.Prelude
+
+newtype MockEnabled = MockEnabled Bool deriving (Show)-- newtype MockEnabled = Bool()
```

Comment:
:nail_care: clear out this comment?

Also new line at the end of this file. I added a vscode extension for this cause I kept forgetting :laughing: